charmcraft.yaml

```diff
@@ -126,6 +126,11 @@ config:
         To create the secret, run the following command: 
         `juju add-secret my-custom-config-ssh-key value=<ssh-key> && juju grant-secret my-custom-config-ssh-key wazuh-server`,
         and use the secret ID output to configure this option.
+    logs-certification-authority:
```

Comment:
maybe we can rename this to `logs-ca-cert` to be more explicit that we're expecting a certificate?